src/EnergyPlus/WeatherManager.cc

```diff
@@ -1419,7 +1419,7 @@ namespace Weather {
                             CurWeekDay -= 7;
                         }
                         WeekDays(i) = CurWeekDay;
-                    } else if ((i >= 4) && (i <= 12)) {
+                    } else {
```

Comment:
[src/EnergyPlus/WeatherManager.cc:1422]:(style),[knownConditionTrueFalse],Condition 'i<=12' is always true